### ISSUE TITLE
Code error

### ISSUE THREAD

**Issue by autoasd** [alexivaner/Deep-Learning-Based-Radio-Signal-Classification#1]
I think your code does not solve the problem of signal modulation classification under low signal-to-noise ratio。
The reason why your code can achieve such high accuracy under low signal-to-noise ratio is because there are duplicate data in the data in your data set.
 code：                                    idx = np.random.choice(range(0,sample_num),size=60000)
The number of 60000 leads to duplication in your data set
When you change 60000 to sample_num and do not load pre-trained weights, your results will be worse.

**Comment by alexivaner** [comment]
> I think your code does not solve the problem of signal modulation classification under low signal-to-noise ratio。
> The reason why your code can achieve such high accuracy under low signal-to-noise ratio is because there are duplicate data in the data in your data set.
> code： idx = np.random.choice(range(0,sample_num),size=60000)
> The number of 60000 leads to duplication in your data set
> When you change 60000 to sample_num and do not load pre-trained weights, your results will be worse.

Hi thank you for your interest in our project, actually, we should admit that in this repository, we only focused on the model and we are not experts in signals. I think that we already use randomized data for our deep learning model. Regarding the pre-trained model, of course, we need to load pre-trained weights,  because that one of the purposes we create a neural network, we want to have trained weight to produce a better result.

**Comment by alexivaner** [comment]
If you want to use another dataset composition, I suggest you train again, but we believe that transformer at least achieves a better result than a conventional way for Low SNR signal.

**Comment by wxw-cyber** [comment]
Why I can't train your model with the full dataset (RML2018.01A), the code always has an ''OOM'' error.
